user/views.py

```diff
@@ -13,6 +13,7 @@ class CreateUserView(generics.CreateAPIView):
 
 class CreateTokenView(ObtainAuthToken):
     renderer_classes = api_settings.DEFAULT_RENDERER_CLASSES
+    serializer_class = AuthTokenSerializer
```

Comment:
The `CreateTokenView` correctly uses `AuthTokenSerializer` to handle user authentication, ensuring that tokens are issued upon successful login.